MSAL/test/app/MSALTestAppSettings.h

```diff
@@ -34,10 +34,9 @@ extern NSString* MSALTestAppCacheChangeNotification;
 @interface MSALTestAppSettings : NSObject
 
 #define MSAL_APP_CLIENT_ID @"clientId"
-#define MSAL_APP_PROFILE @"profile"
+#define MSAL_APP_PROFILE @"currentProfile"
```

Comment:
Why do we need to change it to currentProfile from profile?

Reply:
Earlier NSUserDefaults was saving the profile as a dictionary under the key "profile". I changed the code to save only the profile name instead of the dictionary . For Eg, 
currentProfile : "MSALTest-App". I changed the key name to avoid conflict with the previous dictionary value which was saved.